src/azure-cli/azure/cli/command_modules/acr/_validators.py

```diff
@@ -78,3 +79,10 @@ def validate_task_argument(string, is_secret):
             return {'type': 'Argument', 'name': comps[0], 'value': local_value, 'isSecret': is_secret}
         return {'type': 'Argument', 'name': comps[0], 'value': '', 'isSecret': is_secret}
     return None
+
+
+def validate_retention_days(namespace):
+    days = namespace.days
+    if isinstance(days, int):
```

Comment:
Is this check needed?

Reply:
this is probably needed, or more particularly would be "days is not None"

Reply:
removed.